src/main/java/com/stuypulse/robot/RobotContainer.java

```diff
@@ -49,6 +49,9 @@
     // Autons
     private static SendableChooser<Command> autonChooser = new SendableChooser<>();
 
+    // Alliance Color
+    private static SendableChooser<String> allianceColor = new SendableChooser<>();
+
```

Comment:
move this to inside the color sensor subsystem and change name to something like ALLIANCE_COLOR

Reply:
also i dont think you want a String as the value type for this, especially because the network only sees the key you assign each value. it can be a CurrentBall or a DriverStation.AllianceColor as long as the "null"/"do not use me" case is uploaded as well